feat: Migrate loki pipeline to single entry consumer (#6204)
grafana/alloy: docs/sources/reference/components/loki/loki.enrich.md

| Name                 | Type                  | Description                                                                                      | Default                | Required |
| -------------------- | --------------------- | ------------------------------------------------------------------------------------------------ | ---------------------- | -------- |
| `forward_to`         | `list(LogsReceiver)`  | List of receivers to send enriched logs to.                                                      |                        | yes      |
| `target_match_label` | `string`              | The label from discovered targets to match against, for example, `"__inventory_consul_service"`. |                        | yes      |
| `targets`            | `list(map(string))`   | List of targets from a discovery component.                                                      |                        | yes      |

> | `labels_to_copy`     | `list(string)`        | List of labels to copy from discovered targets to logs. If empty, all labels will be copied.     |                        | no       |

Reviewer: ```suggestion
| `labels_to_copy`     | `list(string)`        | List of labels to copy from discovered targets to logs. If empty, all labels are copied.     |                        | no       |
```

| `logs_match_label`   | `string`              | The label from incoming logs to match against discovered targets, for example `"service_name"`.  |                        | no       |

> If not provided, the `logs_match_label` attribute will default to the value of `target_match_label`.

Reviewer: ```suggestion
If not provided, the `logs_match_label` attribute defaults to the value of `target_match_label`.
```
